pattern-2/bosh-release/jobs/health_check/templates/run.sh.erb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -e
+
+isAlive=0
+
+health_check() {
+    # Check if health check endpoint is alive
+    if curl --output /dev/null --silent --fail -k "$1"
+    then
+        status_code=$(curl --write-out %{http_code} --silent --output /dev/null -k ${1})
+
+        # Check if requests to the health check endpoint produces  200 response
+        if [[ "$status_code" -ne 200 ]] ; then
+            >&2 echo "WSO2 IS $2 produces an invalid response: $status_code"
+            exit 1
+        else
+            echo "WSO2 IS $2 is Running!"
+            isAlive=1
+        fi
+    else
+        >&2 echo "WSO2 IS $2 is not alive. Retrying in 10s..."
+        isAlive=0
+    fi
+}
+
+carbonHealthCheckEP=<%= p("health_check.endpoint") %>
```

Comment:
Change variable name to something like `healthCheckEP` since we are calling different types of endpoints for health checks

Reply:
+1

Reply:
Fixed with 3d673e1